src/renderers/dom/client/syntheticEvents/SyntheticMouseEvent.js

```diff
@@ -63,6 +63,32 @@ var MouseEventInterface = {
       event.pageY :
       event.clientY + ViewportMetrics.currentScrollTop;
   },
+  offsetX: function(event) {
+    return 'offsetX' in event ?
+    event.offsetX :
+    'srcElement' in event ?
+      ((e) => {
+        const target = e.srcElement;
+        const style = target.currentStyle || window.getComputedStyle(target, null);
+        const borderLeftWidth = parseInt(style.borderLeftWidth, 10);
+        const rect = target.getBoundingClientRect();
+        return parseInt((e.clientX - borderLeftWidth - rect.left), 10);
+      })(event) :
+      'undefined';
```

Comment:
@git-richard you're not returning `undefined`, you're returning the string `'undefined'`. Also, if `offset{X,Y}` is added to a `SyntheticMouseEvent` it should be available in all browsers React supports, so a polyfill is required.